cpp/src/Ice/CollocatedRequestHandler.cpp

```diff
@@ -53,13 +54,15 @@ void
 fillInValue(OutputStream* os, int pos, int32_t value)
 {
     const uint8_t* p = reinterpret_cast<const uint8_t*>(&value);
-#ifdef ICE_BIG_ENDIAN
-    reverse_copy(p, p + sizeof(std::int32_t), os->b.begin() + pos);
-#else
-    copy(p, p + sizeof(std::int32_t), os->b.begin() + pos);
-#endif
+    if (endian::native == endian::big)
```

Comment:
Can we use `constexpr` like in the example https://en.cppreference.com/w/cpp/types/endian?